lib/Caxy/HtmlDiff/HtmlDiff.php

```diff
@@ -200,6 +207,12 @@ protected function diffElements($oldText, $newText)
 
         return $wrapStart . $diff->build() . $wrapEnd;
     }
+    
+    protected function diffList($oldText, $newText)
+    {
+        $diff = new ListDiff($oldText, $newText, $this->encoding, $this->isolatedDiffTags, $this->groupDiffs);
```

Comment:
The 4th argument here should actually be `$this->specialCaseTags`, not `$this->isolatedDiffTags` -- the same goes for the call to HtmlDiff on line 206, I missed that in the review :/